impl_core/src/test/java/io/opencensus/implcore/stats/ViewManagerImplTest.java

```diff
@@ -242,11 +242,9 @@ private void testRecordCumulative(Measure measure, Aggregation aggregation, doub
     assertThat(viewData.getView()).isEqualTo(view);
     assertThat(viewData.getWindowData())
         .isEqualTo(CumulativeData.create(Timestamp.create(1, 2), Timestamp.create(3, 4)));
-    StatsTestUtil.assertAggregationMapEquals(
+    assertAggregationMapEquals(
```

Comment:
Why did you modify this class?

Reply:
Looks like intellij did the refactoring when I moved the classes. Undo this change.